### DISCUSSION TITLE
Vumeter flickering and not matching channels on XTouch

### DISCUSSION THREAD

**Discussion by Jagayman** [bjoluc/cubase-mcu-midiremote#47] [Q&A]
I'm monitoring groups only with an xtouch and the vumeters for some channels seem to work "fine" but others flicker as if they are trying to show two different channels at the same time.
Also, related to the above, it seems the vumeters are not tracking to the channel banks -- the scribble strips and the encoders are, but the meters are showing audio and instrument tracks even though I've disabled all but group tracks in the config.
My java skills are not the greatest, but I'll take a look at the code if I have some time and see if I can add more to this discussion.

**Comment by bjoluc** [comment]
Hey @Jagayman, this looks like a legacy MCU remote device configured alongside the MIDI remote script. Can you check if that'S the case (if you haven't solved the whole thing already)?
